scripts/install_anr.sh

```diff
@@ -14,7 +14,7 @@ AVALANCHE_PATH=$(
 # https://github.com/ava-labs/avalanche-network-runner
 GOARCH=$(go env GOARCH)
 GOOS=$(go env GOOS)
-NETWORK_RUNNER_VERSION=1.3.9
```

Comment:
Why is this in the diff?

Reply:
The older ANR versions used the deprecated `genesis` key.